### DISCUSSION TITLE
FCRT energy supply

### DISCUSSION THREAD

**Discussion by badguyland** [hermonochy/Rescueland#21] [Q&A] [answered]
It is unclear what exactly is the energy consumption of a single FCRT. Which different energy sources can FCRTs draw on, what is the maximum endurance, and  how does energy get replenished.

**Comment by hermonochy** [comment, marked as answer]
An FCRT runs on electricity, but it can draw upon almost any energy source, and can last them for about 5 Rescueland year of minor activity eg civillians, or about 1 Rescueland year of vigourous activity, eg constant battles. The energy can be replenished in numerous ways, such as recharging at a recharging station, using natural waste etc.
